.github/workflows/tests.yml

```
@@ -0,0 +1,22 @@
+name: tests
+description: "Run tests (e.g. build) on pull requests"
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)